noseiquela_orm/client.py

```diff
@@ -28,10 +42,19 @@ def save(self, entity, retry=None, timeout=None):
         )
         return entity.key.id
 
+    def _mount_google_entity(self, entity_dict: Dict, key_case: Callable) -> GoogleEntity:
+        entity_key = entity_dict.pop("id")
+        entity = GoogleEntity(entity_key)
+        for key, value in entity_dict.items():
+            key_name = key_case(key)
+            entity[key_name] = value
+
+        return entity
+
```

Comment:
I moved it here (from "`Entity`") because I thought the context made more sense.